src/com/owncloud/android/ui/adapter/FileListListAdapter.java

```diff
@@ -342,6 +367,12 @@ public View getView(int position, View convertView, ViewGroup parent) {
             }
         }
 
+        if (mSelection.get(position) != null) {
+            view.setBackgroundColor(Color.rgb(248, 248, 248));

```

Comment:
fixed in https://github.com/owncloud/android/commit/b1049b25f1fd2fb5ceef8f4acb45f0d8d8f493c1 also moved to a grey standard color, also put that in setup.xml